book/website/reproducible-research/renv/renv-containers.md

```diff
@@ -420,6 +430,120 @@ docker run -d --name dind-rootless --privileged docker:20.10-dind-rootless
+The Docker Daemon is run by root by default and provides a trivial way to escalate privileges and get a high level of access to the hosts devices and filesystem.
+
+(rr-renv-containers-installpodman)=
+### Installing Podman
+
+The Podman documentation has up-to-date instructions for [installing Podman](https://podman.io/getting-started/installation).
+
+It is important to understand that Podman is a tool for running Linux containers and so it requires a Linux host.
+If your computer is running Windows or MacOS, you can use the [Podman remote client](https://github.com/containers/podman/blob/main/docs/tutorials/mac_win_client.md) to interact with Podman on a Linux virtual machine or remote Linux Host.
+
+Alternatively, the MacOS Podman client includes the experimental `podman machine` subcommand for managing a Linux virtual machine that Podman can use.
+Detailed instructions can be found [on Podman's GitHub repository](https://github.com/containers/podman/blob/main/docs/tutorials/mac_experimental.md)
+On Windows you can also run Podman in the [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/) (>= 2.0).
+[This RedHat blog post](https://www.redhat.com/sysadmin/podman-windows-wsl2) has instructions.
+
+For most recent Linux distributions you should [find Podman in the official repositories](https://podman.io/getting-started/installation#linux-distributions).
+Most distributions (including Arch, Debian, Fedora, and Ubuntu) will apply the appropriate configuration to let unprivileged users run Podman automatically.
+If there are any problems, the Podman documentation [has instructions for configuration](https://docs.podman.io/en/latest/markdown/podman.1.html?highlight=rootless#rootless-mode).
+This is as simple as two commands per user who should be able to run Podman.
+
+(rr-renv-containers-commandspodman)=
+### Podman Commands
+
+Podman has a Docker-compatible command line interface so those commands will not be reiterated here.
+The Docker commands in the {ref}`key commands <rr-renv-containers-commands>` should all work by substituting `sudo docker` with `podman`.
+Details of all commands and their options can be found [in the Podman documentation](https://docs.podman.io/en/latest/Commands.html).
+
+(rr-renv-containers-imagespodman)=
+### Building Container Images
+
+Podman can build container images using the `podman build` command.
+Podman will build images from either Dockerfiles or Containerfiles.
+Containerfiles use the same format as Dockerfiles, which are discussed in {ref}`Writing Dockerfiles <rr-renv-containers-dockerfiles>`
+
+(rr-renv-containers-rootlesspodman)=
+### Rootless Containers
+
+Rootless containers are containers run by a normal user (not using `sudo` or with the `root` account).
+These containers never have more privileges than the account that runs them.
+This is an strong security advantage to rootless containers compared to running containers as root or through the Docker daemon.
```

Comment:
Update
"This is an strong security advantage **to** rootless containers compared to running containers as root or through the Docker daemon"
TO
"This is an strong security advantage **for** rootless containers compared to running containers as root or through the Docker daemon"

Reply:
Fixed in f4f92383dcfceea7427138b6c4072f7b4191b235